RecoHI/HiTracking/python/RecoHiTracker_EventContent_cff.py

```diff
@@ -47,6 +49,7 @@
 RecoHiTrackerAOD = cms.PSet(
     outputCommands = cms.untracked.vstring('keep recoTracks_hiGeneralTracks_*_*',
                                            'keep recoTracks_hiGeneralAndPixelTracks_*_*',
-                                           'keep recoVertexs_hiSelectedVertex_*_*'		
+                                           'keep recoVertexs_hiSelectedVertex_*_*',		
+                                           'keep recoVertexs_hiSelectedPixelVertex_*_*'		
```

Comment:
at least for AOD: what is the purpose to save the pre-tracking vertex (hiSelectedPixelVertex)?
Will it be of any use in analyses?